addons/sale/controllers/portal.py

```diff
@@ -166,6 +166,10 @@ def portal_order_page(
             'report_type': 'html',
             'backend_url': backend_url,
             'res_company': order_sudo.company_id,  # Used to display correct company logo
+            'has_active_payment_provider': request.env['payment.provider'].sudo().search([
+                ('state', 'in', ['test', 'enabled']),
+                ('company_id', 'in', request.env.companies.ids),
+            ], limit=1),
```

Comment:
You are adding this in sale and using it in payment :worried: 
(sorry I missed it earlier)